content/en/monitors/monitor_types/forecasts.md

```diff
@@ -15,91 +15,101 @@ further_reading:
+##### Algorithms
 
-## Forecast algorithms
+The available forecast algorithms are `linear` and `seasonal`. 
 
-There are two different forecast algorithms:
+**Linear**<br>
+Use the linear algorithm for metrics that have steady trends but no repeating seasonal pattern. There are three different _models_ which control the linear algorithm's sensitivity to level shifts:
 
-**Linear**: Use this algorithm for metrics that have no repeating seasonal pattern, and tend to have steady trends. On dashboards, linear uses the data within view to create a forecast of the same length. For example, if you set the time selector to "The Past Week", the forecast uses the past week of data to forecast the next week. For monitors, you can explicitly set the amount of history to use, and it is set to one week by default.
+| Model    | Description                                                                                |
+|----------|--------------------------------------------------------------------------------------------|
+| Default  | Adjusts to the most recent trend and extrapolates data while being robust to recent noise. |
+| Simple   | Does a robust linear regression through the entire history.                                |
+| Reactive | Extrapolates recent behavior better at the risk of overfitting to noise, spikes, or dips.  |
 
-{{< img src="monitors/monitor_types/forecasts/linear.png" alt="linear" responsive="true" style="width:80%;" >}}
+**Seasonal**<br>
+Use the seasonal algorithm for metrics with repeating patterns. There are four different _seasonality_ choices:
 
-The linear algorithm has three different _models_ that control how sensitive the algorithm is to level shifts.
+| Option  | Description                                                                                                                                        |
+|---------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| Hourly  | The algorithm expects the same minute after the hour behaves like past minutes after the hour, for example 5:15 behaves like 4:15, 3:15, etc.      |
+| Daily   | The algorithm expects the same time today behaves like past days, for example 5pm today behaves like 5pm yesterday.                                |
+| Weekly  | The algorithm expects that a given day of the week behaves like past days of the week, for example this Tuesday behaves like past Tuesdays.        |
+| Monthly | The algorithm expects that a given day of the month behaves like past days of the month, for example July 1st behaves like June 1st, May 1st, etc. |
 
-The "simple" model does a robust linear regression through the entire history.
+**Note**: This algorithm requires at least two seasons of history and uses up to six seasons for forecasting.
 
-{{< img src="monitors/monitor_types/forecasts/linear_simple.png" alt="linear simple" responsive="true" style="width:80%;">}}
```

Comment:
I think that the most common questions we get from customers about these monitors have to do with choosing the best algorithm/settings to fit their data and I'm a little concerned about the removal of the images here as teaching tools.  I wonder if it makes sense to have a separate page with these images as examples that you can link to here?  For example,  I'd love to have a picture to point out the difference between the `default` and `simple` models